todo_app/app.py

```diff
@@ -15,18 +21,57 @@
 from todo_app.data.View import ViewModel
 from todo_app.flask_config import Config
 
-#app = Flask(__name__)
-#app.config.from_object(Config())
+app = Flask(__name__)
+app.config.from_object(Config())
 
-def create_app():
```

Comment:
Why have we undone this structure? Isn't this going to break our tests?

I assume this is in response to my comments on M3; but I think we can have the best of both worlds here!
If we:
* add back in this function definition
* Indent all the code below this in order to be part of this function
* Add back the `return app` line to end
Then we can have all the existing functionality, plus have the app still pass when tested. E.g. it will look something like:
```
def create_app():
    app = Flask(__name__)
    app.config.from_object(Config())

    # to test hello works
    @app.route('/hello')
    def hello():
        return '...it is working!! .....Hello, World'

   # the rest of the routes here
   # ...
   # ...

    return app
```